### ISSUE TITLE
Installation MusicBot fails on Rpi

### ISSUE THREAD

**Issue by tarqs** [Just-Some-Bots/MusicBot#907]
Hi I'm having problems installing the discord MusicBot
There seems to be a problem when trying to install on Raspbian RPi.

when running
sudo -H pip3.5 install --upgrade -r requirements.txt
it gives cffi-markers sys_platform == win32
doesn't match your platform.

It continues the install with cached versions and finally fails with the message:
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-o0ju21wx/PyNaCl/

Could you help me getting this working?

Thanks in advance!

**Comment by tarqs** [comment]
I'm not sure but I might've fixed this myself. I noticed during trying again and again, that the last command in the guide before you get into python and stuff is a 3 part installation of dependancies. 
1. Libopus-dev
2. Libffi-dev
3. Libsodium-dev

It seems that the 3rd is not found by the raspberry and it breaks the entire installation of the whole thing. If you break these apart by doing:
sudo apt-get install libopus-dev
sudo apt-get install libffi-dev
sudo apt-get install libsodium-dev

It will only fail the libsodium and still install the first 2, whereas if you do the command in 1 go like so: 
sudo apt-get install libopus-dev libffi-dev libsodium-dev
it will skip over the first 2 because it can't find the libsodium-dev repository / depenancy.

Later when trying to run the musicbot (after I broke the lib-dev depenancies apart it at least got further in firing up the bot. To my great surprise, because I haven't gotten around to building permissions.ini + options.ini properly


Will post back once I get everything running properly
